### DISCUSSION TITLE
ΓΕ2 - Μέρος Α - Συνεννόηση για την επιλογή εικόνων

### DISCUSSION THREAD

**Discussion by afoscolou** [sdy60-2022/.github#38] [Ideas]
Σύμφωνα με τις οδηγίες πρέπει να γίνει κάποιος συντονισμός μεταξύ μας ώστε να μην ανεβάσουμε τις ίδιες εικόνες. Μπορεί να είναι παρεμφερείς, όπως και το θέμα, αλλά καλύτερα να υπάρξει μια προ-συνεννόηση.  Προτείνω όποιος βρίσκει τι θέλει να χρησιμοποιήσει, να το ποστάρει σε αυτή την συζήτηση για να μην κάνουμε διπλό κόπο.

**Comment by Katethoma** [comment]
Καλησπέρα. Εγώ θα ασχοληθώ με συσκευές της Apple

**Reply by dimidimots** [reply]
Καλησπέρα εγώ σκέφτομαι να ασχοληθώ με την γνωστή συνεργατική εγκυκλοπαίδεια.

**Comment by vkaragia** [comment]
Καλησπέρα σας, εγώ θα ασχοληθώ με στρατιωτικές εφαρμογές.

**Comment by sotkot** [comment]
Καλησπέρα σας,

Εγώ θα ασχοληθώ με το Microsoft HoloLens και το Google Glass.

**Comment by afoscolou** [comment]
Καλησπέρα.

Θα γράψω για το Nokia 9500 και Nokia 9210 Communicator

**Comment by stdi** [comment]
Γεια σας!

Θα ήθελα να ασχοληθώ με τα Oculus VR headsets/store.

**Comment by NikosAndreadakis** [comment]
Adaptive cruise control & Camera lane assist.

**Comment by gregtsag** [comment]
ARQuake και Interaction in a collaborative augmented reality environment (cake platter)

**Comment by atsiano81** [comment]
Google Maps Street view - Live view και htc vive

**Comment by vangelidis** [comment]
Aura Interactor και Teslasuit

**Comment by gsopy** [comment]
View-Master και NASA VIEW Headset

**Comment by vagelisNerantzis** [comment]
Magic Leap One και SixthSense
